packages/react-composites/tests/browser/common/fixtureHelpers.ts

```diff
@@ -153,41 +153,43 @@ export const createMeetingUsers =
 /**
  * Load a URL in the browser page.
  * @param browser Browser to create Page in.
- * @param serverUrl URL to a running test app.
- * @param user User to load url for.
+ * @param url URL to a running test app.
  * @returns
  */
-const loadUrlInPage = async (page: Page, serverUrl: string, user: ChatUserType | CallUserType): Promise<Page> => {
+export const loadNewPage = async (browser: Browser, url: string): Promise<Page> => {
+  const page = await browser.newPage();
   await page.setViewportSize(PAGE_VIEWPORT);
-
-  const url = buildUrl(serverUrl, user);
-  await page.goto(url, { waitUntil: 'load' });
```

Comment:
Are we all good without option: waitUntil: 'load'
Is it a default option of page.goto ?

Reply:
Yeah `load` is the default